auth-backend/src/middleware/jwt-token-validator.ts

```
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from 'express';
+import { validateRefreshToken } from '../utils/helpers';
+
+export const validateAuthTokens = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    //@ts-ignore
+    const [_, refresh] = req!.headers!.authorization?.split(' ,');
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)